packages/lib/gpc/test/gpcChecks.spec.ts

```diff
@@ -49,7 +49,7 @@ describe("Entry bounds check config check should work", () => {
           inRange: { min: 3n, max: 100n },
           notInRange: { min: 204n, max: 900n }
         },
-        1
+        2
```

Comment:
I think this isn't right.  I think this indicates that check is determining a requirement which is too high, because it's not canonicalizing yet.  But in real proof generation, canonicalization is always done after checking.  So if these constraints can be reduced down to 1, the proof only needs.

Net result: I think the check phase needs to canonicalize before counting.  For this case I think it's fine to canonicalize in a temporary way and throw away the result, rather than trying to mess with the overall order of check vs. canonicalize in the compiler.